content/docs/demos/prometheus_grafana.md

```diff
@@ -1,49 +1,61 @@
 ---
-title: "Prometheus & Grafana"
-description: "Describes basic steps to get up and running a Prometheus and Grafana stack with OSM specific configuration and Dashboards"
+title: "Integrate OSM with Prometheus and Grafana"
+description: "Describes how to set up an OSM-specific configuration and dashboards with your own Prometheus and Grafana stack"
 type: docs
 weight: 25
 ---
 
-# OSM's Prometheus and Grafana stack
+# Integrate OSM with your own Prometheus and Grafana stack
 
-## Overview
-The following doc walks through the process of creating a simple Prometheus and Grafana stack to enable observability and monitoring of OSM's service mesh.
+The following article shows you how to create an example bring your own (BYO) Prometheus and Grafana stack on your cluster and configure that stack for observability and monitoring of OSM. For an example using an automatic provisioning of a Prometheus and Grafana stack with OSM, see the [Observability](https://docs.openservicemesh.io/docs/getting_started/observability/) getting started guide.
 
-This setup is not meant for production, for production-grade deployments one should refer to [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md).
+The configuration created in this article should not be used in production environments. For production-grade deployments, see [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md) and [Deploy Grafana in Kubernetes](https://grafana.com/docs/grafana/latest/installation/kubernetes/).
+
+## Deploy an example Prometheus instance
+
+Use `helm` to deploy a Prometheus instance to your cluster.
 
-## Deploying a Prometheus Instance
-The easiest way to deploy a Prometheus instance is through Helm:
 ```
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
 helm install stable prometheus-community/prometheus
```

Comment:
Should we call out that we are using the default namespace?

Reply:
fixed